math/mathmore/inc/Math/SpecFuncMathMore.h

```diff
@@ -121,7 +121,9 @@ namespace Math {
   double assoc_legendre(unsigned l, unsigned m, double x);
 
 
-
+  namespace internal{
```

Comment:
We need to leave a comment why this exists. People will wonder why there is this function definition without any info what this does.